docs/development/setup-vagrant.md

```diff
@@ -903,11 +900,11 @@ vagrant ssh -- 'sudo modinfo -F version vboxsf'
 
 The bug has not been fixed upstream as of this writing, but you may be
 able to work around it by downgrading VirtualBox Guest Additions to
-6.0.4. To do this, create a `~/.zulip-vagrant-config` file and add
+5.2.44. To do this, create a `~/.zulip-vagrant-config` file and add
 this line:
 
 ```text
-VBOXADD_VERSION 6.0.4
+VBOXADD_VERSION 5.2.44
```

Comment:
Why does this version go down?

Reply:
The [ETXTBSY bug](https://www.virtualbox.org/ticket/19004) was introduced in VirtualBox Guest Additions 6.0.6 (2019-04-17), and Guest Additions 6.0.4 (2019-01-28) no longer builds on kernel 5.4.

Although Guest Additions 6.0.16 (2020-01-14) gained kernel 5.4 support, we can’t use it due to the still-unfixed ETXTBSY bug.

Meanwhile, maintenance continued on the VirtualBox 5.2.x branch. It was never affected by ETXTBSY, but it also gained kernel 5.4 support in Guest Additions 5.2.36 (2020-01-14). So we can use the final 5.2.x release, which is Guest Additions 5.4.44 (2020-07-14).

This is the last kernel upgrade we can buy this way; unless they fix ETXTBSY, we’ll have to abandon VirtualBox on Windows for the next cycle. That’s not the end of the world since we have the WSL2 setup, even though its integration is messier. But we might as well put it off while we can.

Reply:
OK, thanks for the explanation!  I agree with the analysis that we could drop Virtualbox on Windows but it'd be nice not to.